libraries/botbuilder/src/cloudAdapter.ts

```diff
@@ -145,6 +144,35 @@ export class CloudAdapter extends CloudAdapterBase implements BotFrameworkHttpAd
         }
     }
 
+    /**
+     * Exposes the same functionality as [BotFrameworkAdapter.processActivityDirect](https://github.com/microsoft/botbuilder-js/blob/main/libraries/botbuilder/src/botFrameworkAdapter.ts).
```

Comment:
I don't think we should reference a deprecated method in the documentation of a new method. We should explain what the method does. In any case, we could include a comment in the old method referencing this one.